### PR TITLE
have a csv generation file and a postgres script

### PR DESCRIPTION
**Pull request by songkenny**
was able to load a smaller file (100 pid)

### CODE REVIEW

File: databaseSDC/indexPost.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+})
+pool.query('SELECT NOW()', (err, res) => {
+  console.log(err, res)
+  pool.end()
+})
+const client = new Client({
+  user: 'dbuser',
+  host: 'database.server.com',
+  database: 'mydb',
+  password: 'secretpassword',
```
Comment (karebae) [inline databaseSDC/indexPost.js:17]: nice

Comment (songkenny) [reply databaseSDC/indexPost.js:17]: LOL whoops you can ignore that part

File: seed/seedling.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      }
+      characteristics = JSON.stringify(characteristics);
+      characteristics = characteristics.replace(/"/gi, '"""');
+
+      const numOfSkus = faker.random.number({ min: 1, max: 8 });
+      const skusArr = [];
+      for (let i = 0; i < numOfSkus; i++) {
+        skusArr.push(faker.commerce.color());
+      }
+      let skus = JSON.stringify(skusArr);
```
Comment (karebae) [inline seed/seedling.js:47]: this is clever!!

**Review by karebae — APPROVED** [review]
looks good! your indexPost.js is very interesting~ good use of clever logic and good job with your schemas! looks very comprehensive

**Pull request merged**